Kernel/Devices/Storage/NVMe/NVMeDefinitions.h

```diff
@@ -34,10 +34,20 @@ struct IdentifyNamespace {
     u64 rsvd3[488];
 };
 
+// FIXME: For now only one value is used. Once we start using
+//  more values from id_ctrl command, use separate member variables
+// instead of using an array.
+struct IdentifyController {
+    u16 arr[2048];
```

Comment:
Can we have a more descriptive name than `arr`?

Reply:
Added a struct entry for the `oacs` instead of using offset in an array to determine the value. Thanks.